policies/optimized-images.md

```diff
@@ -216,16 +277,6 @@ When a document is disallowed to use `unoptimized-images` policy, its `<img>` el
   </tr>
 </table>
 
-For an `<img>` element, if its file size is within the limit, the image will be rendered correctly; otherwise the image will be rendered as placeholder images.
-
-
-**Future Development**
-
-Image formats affect file size. We want to support different default values for different image formats.
-We want to allow developers to specify the parameters as well. In practice, they would look something like this:
-```html    
-<iframe allow="unoptimized-images(image/bmp(1024, 0.5), image/jpeg(2048, 0.2))"></iframe>
-```
-Note: any otherwise unspecified MIME types will be using the default values. 
+For an `<img>` element, if its file size is within the allowrance, the image will be rendered correctly; otherwise the image will be rendered as placeholder images.
```

Comment:
Suggestion:

Any `<img>` element whose file size is within the allowance will be rendered correctly;